index.html

```diff
@@ -4,8 +4,53 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="css/style.css" />
+
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/images/icons/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/images/icons/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/images/icons/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/images/icons/site.webmanifest" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="src/style.css" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <div class="menu-icon"></div>
+      </header>
+      <main class="main">
+        <h1 class="title">Charlie Giorgania</h1>
+        <h2 class="subtitle">
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Taka a look of my works. If you like what you see
+          and have a project you need coded, don’t hesitate and contact me.
+        </h2>
```

Comment:
- [ ] Remember to keep your HTML semantic, this should be wrapped in a `<p>` tag instead of a `<h2>` tag. You can read more about [paragraphs here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p) and [h tags here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).